lib/convection/model/template/resource/aws_elbv2_load_balancer.rb

```diff
@@ -0,0 +1,38 @@
+require_relative '../resource'
+
+module Convection
+  module Model
+    class Template
+      class Resource
+        ##
+        # AWS::ElasticLoadBalancingV2::LoadBalancer
+        ##
+        class ELBV2LoadBalancer < Resource
+          include Model::Mixin::Taggable
+
+          type 'AWS::ElasticLoadBalancingV2::LoadBalancer', :elbv2_load_balancer
+          property :load_balancer_attributes, 'LoadBalancerAttributes', :type => :list
+          property :name, 'Name'
+          property :scheme, 'Scheme'
+          property :security_group, 'SecurityGroups', :type => :list
+          alias_method :security_groups, :security_group
```

Comment:
I'm curious why we want to be inconsistent with what amazon defined with cloudformation?  Looking at the docs:
http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-elasticloadbalancingv2-loadbalancer.html
They define it as "SecurityGroups" because it's a list not a single thing, which makes sense.  We could then add a singular function to add a single security group, which would make sense but might be a breaking change.